packages/transport/src/modules/index.ts

```diff
@@ -65,6 +65,9 @@ export * from "./messages/transmission/MessageEnvelope";
 export * from "./messages/transmission/MessageEnvelopeRecipient";
 export * from "./messages/transmission/MessageSignature";
 export * from "./messages/transmission/MessageSigned";
+export * from "./publicRelationshipTemplatesReference/backbone/PublicRelationshipTemplateReferenceClient";
```

Comment:
A plural "s" is in the wrong position (we need to write "References" instead of "Templates"), please change the according folder name.